app/controllers/application_controller.rb

```diff
@@ -9,7 +10,8 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << :username
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:username, :email, :password,
```

Comment:
Line is too long. [89/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.